model/trace/database.yaml

```diff
@@ -36,16 +36,6 @@ groups:
         tag: connection-level
         requirement_level:
           conditionally_required: If using a port other than the default port for this DBMS and if `server.address` is set.
-      - ref: network.peer.address
```

Comment:
TODO: also update examples

Reply:
Will you update them as part of this PR or as a follow up?

Reply:
In this PR, I'd like to get a blessing from DB WG first anyway.